src/main/java/Task.java

```diff
@@ -12,7 +12,14 @@ public void setIsDone(boolean newIsDone) {
     }
 
     public String getFullTask() {
-        String isDoneSymbol = isDone ? "\u2713" : "\u2717";
+        /** For automated testing, replace isDoneSymbol with the following line,
+         * String isDoneSymbol = isDone ? "✓" : "✗";
+         *
+         * For production, replace isDoneSymbol with the following line,
+         * String isDoneSymbol = isDone ? "\u2713" : "\u2717";
+         */
+
+        String isDoneSymbol = isDone ? "✓" : "✗";
```

Comment:
make the symbol more clear and let people know what will be printed out clearly

Reply:
May I have some suggestion on how I should approach this?

Reply:
sorry I mean you did well~ You've made the symbol more clear